### PR TITLE
chore: update cliffy to 0.10.0

### PR DESCRIPTION
**Pull request by SteelAlloy**

### CODE REVIEW

**Comment by t8** [comment]
This portion of `deps.ts` is still on Cliffy v0.9.0:

```ts
export {
  Input,
  Confirm,
  List,
} from "https://deno.land/x/cliffy@v0.9.0/prompt.ts";
```

File: deps.ts [hunk 1]
```diff
@@ -26,7 +26,11 @@ export {
   stringify,
 } from "https://deno.land/std@0.56.0/encoding/yaml.ts";
 
-export { Command } from "https://deno.land/x/cliffy@v0.9.0/command.ts";
+export {
+  Command,
+  HelpCommand,
+  CompletionsCommand,
+} from "https://deno.land/x/cliffy@v0.10.0/command.ts";
```
Comment (t8) [inline deps.ts:33]:
```suggestion
} from "https://x.nest.land/cliffy@0.10.0/command.ts";
```
😉

**Review by t8 — CHANGES_REQUESTED** [review]
Everything looks good except for that other cliffy import and the fact that you aren't importing Cliffy `0.10.0` from nest.land. Is it meant to be kept at 0.9.0?

File: deps.ts (hunk 1 above)
Comment (SteelAlloy) [reply deps.ts:33]: Not at all, I'll change that right away

**Comment by notfilippo** [comment]
On hold until https://github.com/c4spar/deno-cliffy/pull/45 gets merged.

**Pull request merged**